clients/github-graphql/src/main/kotlin/GitHubService.kt

```diff
@@ -73,7 +72,7 @@ class GitHubService private constructor(
          * Optionally, the [url] for the GitHub GraphQL endpoint can be configured.
          */
         fun create(token: String, url: URI = ENDPOINT): GitHubService {
-            val client = HttpClient(engineFactory = OkHttp) {
+            val client = HttpClient {
```

Comment:
What's the effect of this change, does it still use the OkHttp client or the Ktor client now?